pkg/servicecentral/types.go

```diff
@@ -0,0 +1,140 @@
+package servicecentral
+
+import "time"
+
+type ServiceOwner struct {
+	Username string `json:"username"`
+}
+
+type ServiceData struct {
+	ServiceUUID        *string      `json:"service_uuid,omitempty"`
+	CreationTimestamp  *string      `json:"creation_timestamp,omitempty"`
+	ServiceName        string       `json:"service_name,omitempty"`
+	ServiceOwner       ServiceOwner `json:"service_owner,omitempty"`
+	ServiceTier        int          `json:"service_tier,omitempty"`
+	Tags               []string     `json:"tags,omitempty"`
+	Platform           string       `json:"platform,omitempty"`
+	Misc               []miscData   `json:"misc,omitempty"`
+	PagerDutyServiceID string       `json:"pagerduty_service_id,omitempty"`
```

Comment:
Not sure if this should be public? We will add OpsGenie later as well?

Reply:
I believe we'll just have the OpsGenie team name as an optional `string`, but best to double-check :)